client/gui/console.hpp

```diff
@@ -53,9 +53,9 @@ namespace tec {
 
 	private:
 		// TODO Store a RingBuffer<tuple<color, text> instead of raw text ?
-		tec::RingBuffer< std::tuple< ImVec4, std::string >, 4096> buf;
+		std::shared_ptr<tec::RingBuffer< std::tuple< ImVec4, std::string >, 4096>> buf;
```

Comment:
Does this need to be a shared ptr?

Reply:
The allocation was huge, so I just defaulted to shared. We can certainly look if it can be a unique